src/commands/functions/createTag.ts

```diff
@@ -64,10 +65,32 @@ export async function createTag(item?: TagsTreeItem | Uri): Promise<void> {
         if (command === undefined) {
             return;
         }
+
+        // there is a  special way to publishing a tag for .net5.0
+        const envPick = await window.showQuickPick(
+            [".net5.0", "other"],
+            { placeHolder: "Select runtime" }
+        );
```

Comment:
Is there a good way to detect automatically if they are using .NET? If so, we might be able to skip this prompt.

Reply:
Yes we could. We could chek if there is a *.csproj in the folder. Then the prject is using .NET
Now project can be using Net3.1 or Net5.0, there we would need to read the *.csproj and find out what version it is from there :(